components/prefs/testing_pref_store.cc

```diff
@@ -53,9 +53,6 @@ bool TestingPrefStore::IsInitializationComplete() const {
 void TestingPrefStore::SetValue(const std::string& key,
                                 base::Value value,
                                 uint32_t flags) {
-#ifndef USE_HACKY_COBALT_CHANGES
-  DCHECK(!value.is_none());
```

Comment:
Do we want to completely remove these DCHECKs?

Reply:
In the m114 upstream code, this DCHECK is removed.